yt_dlp/cookies.py

```diff
@@ -307,6 +308,12 @@ def _extract_chrome_cookies(browser_name, profile, keyring, logger):
             counts['unencrypted'] = unencrypted_cookies
             logger.debug(f'cookie version breakdown: {counts}')
             return jar
+        except PermissionError as error:
+            if compat_os_name == 'nt' and error.errno == 13:
+                message = 'Could not copy Chrome cookie database. See  https://github.com/yt-dlp/yt-dlp/issues/7271  for more info'
+                logger.error(message)
+                raise DownloadError(message)  # force exit
```

Comment:
Do we need to exit here?

Reply:
If we dont then there will be a traceback obscuring the error message. Its not required but its cleaner to exit.

Reply:
What about `logger.error(tb=False)`?

Reply:
That doesn't really work, since we have to raise the error to break out of the cached property on `YoutubeDL`. That means we get a traceback (and a "Uncought error in __main__" if using binary)